### DISCUSSION TITLE
Changing viscosity, surface tension in Invasion Percolation algorithm

### DISCUSSION THREAD

**Discussion by debanik1** [PMEAL/OpenPNM#2257] [Q&A] [answered]
How do I change the viscosity or the surface tension in the capillary dominated [Invasion Percolation](https://github.com/PMEAL/OpenPNM/tree/release/examples/notebooks/algorithms/percolation) algorithm? I just want to see what the different values do to the output.

**Comment by jgostick** [comment, marked as answer]
You change them on the phase object of the invasion fluid, for instance: ``water['throat.surface_tension'] = 0.500``.  Viscosity will have no effect since it's capillary dominated, the algorithm doesn't look at it.

**Comment by debanik1** [comment]
What's the significance of the color codes in the plots? How do I refer?
